src/transaction/components/Transaction.tsx

```diff
@@ -35,7 +38,9 @@ export function Transaction({
       onStatus={onStatus}
       onSuccess={onSuccess}
     >
-      <div className={cn(className, 'flex w-full flex-col gap-2')}>
+      <div
+        className={cn(componentTheme, className, 'flex w-full flex-col gap-2')}
```

Comment:
`className` should be last here so it can override other styles

Reply:
Good eye, will fix in the follow up!